src/fundus/publishers/de/krautreporter.py

```diff
@@ -0,0 +1,46 @@
+import json
+from datetime import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute, utility
+
+
+class KrautreporterParser(ParserProxy):
+    class V1(BaseParser):
+        _summary_selector = CSSSelector("p.article-headers-standard__teaser")
+        _subheadline_selector = CSSSelector("div.article-markdown > h2")
+        _paragraph_selector = CSSSelector("div.article-markdown > p")
```

Comment:
In most articles the last line is often something like _Redaktion: Brigitte Wenger, Schlussredaktion: Susan Mücke, Bildredaktion: Gabriel Schaefer; Audioversion: Iris Hochberger_ 
I would consider filtering those out. I think it should be relatively easy since the p elements do not have any own text and only contain one em element as a child. At least with XPath it should simple with something like  `p[text()]` or `p[not(em)]`, although the latter may be riskier.

Reply:
I think this is dangerous, as I’ve seen articles with paragraphs that contain only text wrapped in an em-tag. 
We could maybe filter out those that are the last child of the article-markdown div and only contain an em-tag?

Reply:
Sounds good 👍

Reply:
Found a solution that builds on the assumption that those paragraphs are proceeded by `hr` tags.

```XPath
//div[contains(@class, 'article-markdown')] /p[not(preceding::hr)]
```

Reply:
Okay, it seems that this assumption is also false 😅 Looks like we have to use regex.

Reply:
I adjusted the selector a bit so now every `p` without text directly following an `hr` element isn't selected. There is still a bit of risk but I think that might be our best option besides returning to regexes.

```
//div[contains(@class, 'article-markdown')] /p[text() or not(preceding-sibling::*[1][self::hr])]
```